addon/components/basic-header.js

```diff
@@ -14,6 +14,13 @@ export default Ember.Component.extend({
   column: null,
   resizable: computed.readOnly('column.resizable'),
 
+  didReceiveAttrs() {
+    this._super(...arguments);
+
+    let columns = this.get('parentView');
+    run.scheduleOnce('actions', columns, columns._reflowStickyHeaders);
```

Comment:
:+1: Good catch. We should make this non-private if we're calling it here though. Maybe also make it handle the `scheduleOnce`? That would allow us to remove some repeated calls to `scheduleOnce` in `table-columns` as well.